build/build.go

```diff
@@ -26,3 +26,13 @@ func Runfile() string {
 	// return converted output value
 	return output
 }
+
+// Native function to call Linux command
```

Comment:
A great way to think about doc comments for functions is to answer the following three questions (in the form of a sentence where the function name is the subject):

1. What arguments does it take?
2. What does it do with them?
3. What does it return?

For example, here we might write something like:

```go
// Native runs a fixed command via the Unix shell, and returns the string output.
```

In fact, I wouldn't even worry about doc comments at this stage, because the API design is still very fluid. The right time to start writing doc comments, I think, is around the time you're thinking of doing a final review and getting ready to merge your code to the main line.

Reply:
thanks for the note on how to write good function comments, I started noting these here for my future reference: https://trello.com/c/heOJkmq3

Okay, I'll skip commenting functions for now - but will otherwise keep adding for reference.